doc/source/cluster/kubernetes/getting-started/rayjob-quick-start.md

```diff
@@ -147,3 +147,9 @@ helm uninstall kuberay-operator
 # Step 6.3: Delete the Kubernetes cluster
 kind delete cluster
 ```
+
+## Advanced Usage
+
+The Pod template for the Kubernetes Job that runs `ray job submit` can be customized by setting the `submitterPodTemplate` field in the RayJob custom resource.  See <https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray_v1alpha1_rayjob.yaml> for an example (commented out in this file).
+
+If `submitterPodTemplate` is unspecified, the pod will consist of a container named `ray-job-submitter` with image matching that of the Ray head, resource requests of 500m CPU and 200MiB memory, and limits of 1 CPU and 1GiB memory.
```

Comment:
```suggestion
If `submitterPodTemplate` is unspecified, the Pod will consist of a container named `ray-job-submitter` with image matching that of the Ray head, resource requests of 500m CPU and 200MiB memory, and limits of 1 CPU and 1GiB memory.
```

Reply:
Would you mind adding a link to your benchmark result?

Reply:
The benchmark was kind of informal so I don't know if it makes sense to include it in the user-facing documentation. Would you be okay with merging this PR without the benchmark? If you like I can add the link to the benchmark in the KubeRay repo as a code comment.

Reply:
Both are fine.